examples/example_files/off_files/testOFF.top

```diff
@@ -0,0 +1,315 @@
+TITLE
+
+	name: 	 hill_formula: C6H12
+	----------------------------------------
+	| created from OpenForceField topology |
+	| use Amber Block for OpenFF topology! |
+	----------------------------------------
+
+	 >>> Generated with python lib function_libs utilities. (riniker group)
+	 >>> line_seperator: '\n'	 field_seperator: '\t'	 comments_char: '#'
+END
+PHYSICALCONSTANTS
+# FPEPSI: 1.0/(4.0*PI*EPS0) (EPS0 is the permittivity of vacuum)
+138.9354
+# HBAR: Planck's constant HBAR = H/(2* PI)
+0.0635078
+# SPDL: Speed of light (nm/ps)
+299792.458
+# BOLTZ: Boltzmann's constant kB
+0.00831441
+END
+TOPVERSION
+2.0
+END
+ATOMTYPENAME
+2
+n16
+n2
+END
+RESNAME
+1
+C6H12
+END
+SOLUTEATOM
+#   NRP: number of solute atoms
+ 	 18
+#  ATNM: atom number
+#  MRES: residue number
+#  PANM: atom name of solute atom
+#   IAC: integer (van der Waals) atom type code
+#  MASS: mass of solute atom
+#    CG: charge of solute atom
+#   CGC: charge group code (0 or 1)
+#   INE: number of excluded atoms
+# INE14: number of 1-4 interactions
+# ATNM MRES PANM IAC     MASS       CG  CGC INE
+#                                           INE14
+	1	1	C1	1	12.01078	-0.0764	0	10	2	3	5	6	7	8
+											9	10	17	18
+										5	4	11	12	15	16
+	2	1	C2	1	12.01078	-0.0764	0	9	3	4	6	7	8	9
+											10	11	12
+										5	5	13	14	17	18
+	3	1	C3	1	12.01078	-0.0764	0	8	4	5	9	10	11	12
+											13	14
+										5	6	7	8	15	16
+	4	1	C4	1	12.01078	-0.0764	0	8	5	6	11	12	13	14
+											15	16
+										4	9	10	17	18
+	5	1	C5	1	12.01078	-0.0764	0	7	6	13	14	15	16	17
+											18
+										4	7	8	11	12
+	6	1	C6	1	12.01078	-0.0764	0	6	7	8	15	16	17	18
+										4	9	10	13	14
+	7	1	H1	2	1.007947	0.0382	0	1	8
+										4	9	10	17	18
+	8	1	H2	2	1.007947	0.0382	0	0
+										4	9	10	17	18
+	9	1	H3	2	1.007947	0.0382	0	1	10
+										2	11	12
+	10	1	H4	2	1.007947	0.0382	0	0
+										2	11	12
+	11	1	H5	2	1.007947	0.0382	0	1	12
+										2	13	14
+	12	1	H6	2	1.007947	0.0382	0	0
+										2	13	14
+	13	1	H7	2	1.007947	0.0382	0	1	14
+										2	15	16
+	14	1	H8	2	1.007947	0.0382	0	0
+										2	15	16
+	15	1	H9	2	1.007947	0.0382	0	1	16
+										2	17	18
+	16	1	H10	2	1.007947	0.0382	0	0
+										2	17	18
+	17	1	H11	2	1.007947	0.0382	0	1	18
+										0
+	18	1	H12	2	1.007947	0.0382	1	0
+										0
+END
+BONDSTRETCHTYPE
+# 	 NBTY: number of covalent bond types
+ 	 2
+# 	 CB 	 CHB 	 B0
+	4.78018e+06	2.21435e+05	1.52190e-01
```

Comment:
these are still a little different from my amber2gromos topology, e.g. 5.325e+06 instead of 4.780e+06, but I assume these are just FF differences

Reply:
Yes, they are identical to off->amber->gromos